recipes/lldb/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "5.0" %}
+{% set sha256 = "d60b7151ae41a496132e28f1b40fc068c3a3ca99d729a0331235eac8987117f5" %}
+{% set commit = "c9b520a36641d09f00c7ee079353a1af19e4c707" %}
+
+package:
+  name: lldb
+  version: {{ version }}
+
+source:
+  fn: lldb-{{ version }}.src.tar.xz
+  url: https://github.com/llvm-mirror/lldb/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<36 or osx]
+  features:
+    - vc14    # [win and py36]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.4.3
+    
+    - ninja  # [win32 or win64]
+
+    - system  # [linux and not armv7l]
+
+    # "compiling .pyc files" fails without this:
+    - python
+
+    - llvmdev =={{ version }}
+    - clangdev =={{ version }}
+    - libxml2 2.9.*
+    - swig
+    - libedit  # [linux]
+
+  run:
+    - vs2015_runtime  # [win]
+
+    - llvmdev =={{ version }}
+    - clangdev =={{ version }}
+    - libedit  # [linux]
```

Comment:
Why is this needed? 😕

Reply:
Dynamic loader failed to resolve a symbol at runtim.e

Reply:
What symbol? Do you have a log?

Reply:
libedit is a requirement for linux. See https://lldb.llvm.org/build.html